env/include/net/packet.h

```diff
@@ -126,5 +126,5 @@ static inline void net_packet_checksum_update(struct net_ipv4_header* ipv4_heade
 static inline void net_packet_checksum_update_32(struct net_ipv4_header* ipv4_header, uint32_t old_word, uint32_t new_word, bool in_ip)
 {
 	net_packet_checksum_update(ipv4_header, (uint16_t) old_word, (uint16_t) new_word, in_ip);
-	net_packet_checksum_update(ipv4_header, (uint16_t)(old_word >> 16), (uint16_t)(new_word >> 16), in_ip);
+	net_packet_checksum_update(ipv4_header, (uint16_t) (old_word >> 16), (uint16_t) (new_word >> 16), in_ip);
```

Comment:
is this using the existing clang-format I had added to the repo, btw? did I just forget to re-run it, or is it something else?

Reply:
yup, I used the supplied conf, it seems that indeed you forgot to ran it (and that's why CI are good) :)